src/OrchardCore/OrchardCore/Shell/Distributed/DistributedShellHostedService.cs

```diff
@@ -540,5 +543,26 @@ private async Task<bool> TryWaitAsync(TimeSpan delay, CancellationToken stopping
                 return false;
             }
         }
+
+        private bool _isDisposed;
```

Comment:
Maybe define it above with other private fields.

Can we use `_disposed`? As it is mainly used in aspnetcore, much more than `_isDisposed`, otherwise that's okay ;)

Then maybe use the same name everywhere, if you want.